sql/pg_net.sql

```diff
@@ -232,7 +276,7 @@ create type net.http_response_result as (
 
 
 -- Collect respones of an http request
--- API: Public
+-- API: Private
```

Comment:
Should we also append `_` to the function name?

Reply:
Right, done.